sass/compat/themes/leadengine/_tom-select-control.scss

```diff
@@ -0,0 +1,16 @@
+// ------------------------------------
+// TOM SELECT CONTROL
+// ------------------------------------
+
+body#{ $_body-theme-selector } .ts-wrapper.single .ts-control {
+	z-index: z( 'above' );
+	height: remsize( 5rem );
+	padding-left: remsize( 2rem );
```

Comment:
@luiggidev,
- when `left` or `right` is needed on the properties name or value, we need to use the variables `$align-left` and `$align-right` respectively so that the RTL files are generated correctly mirroring the layout.
- This also applies when properties for both `left` or `right` are used, and also when they have the same value.
- If using specifically `left` or `right` is required regardless of LTR/RTL, then an inline comment is required.